packages/semcom-components/vite.build.config.ts

```diff
@@ -0,0 +1,18 @@
+import path from 'path';
```

Comment:
Why split this over multiple files? A single config, conditional on command and/or mode, is suggested by the [Vite docs](https://vitejs.dev/config/#conditional-config).